index.html

```diff
@@ -250,7 +269,7 @@ <h3>Get my Resume</h3>
         <img src="img/Rectangle%2015.png" alt="" id="resume-box"> /* use as css */
         <span class="material-icons">open_in_new</span>
         <div class="social-icons">
-            <img src="img/stackoverflow-24.png" alt="">
+            <img src="img/ic_stackoverflow.svg" alt="">
             <img src="img/github.svg" alt="" id="github">
             <img src="img/twitter.svg" alt="" id="twitter">
             <img src="img/linkedin.svg" alt="" id="linkedin">
```

Comment:
- [ ] Please decrease the size of icons on the footer, currently the render huge:

Your design  | Figma design
------------- | ------------- 
<img width="1426" alt="Screenshot 2022-03-31 at 16 44 07" src="https://user-images.githubusercontent.com/77038610/161057525-20330865-8c9f-4eac-a885-92a3c754e2e3.png"> | <img width="678" alt="Screenshot 2022-03-31 at 16 43 52" src="https://user-images.githubusercontent.com/77038610/161057491-bcdbfec9-b401-4401-b54f-ace78088734d.png">

- [ ] On the right side above `get my cv` I can see text  /* use as css */ please delete that
- [ ] After you decrease the side of icons, please make sure that you have 6 icons as provided in Figma

Reply:
Reduced size of the social-icons
<img width="328" alt="Screenshot 2022-04-01 at 07 55 54" src="https://user-images.githubusercontent.com/40231363/161199468-6f14e07f-934f-47ba-a455-ab35215ec8bb.png">
s